kagiso_smart_404/utils.py

```diff
@@ -1,17 +1,50 @@
 from wagtail.wagtailcore.models import Page
 
 
+def page_slug(slug):
+    # Full slug might be /news/sports/somesportsarticle/
+    # The page slug is therefore `somesportsarticle`
+    slugs = slug.split('/')
+
+    # Remove all empty strings
+    # '/news/article' => ['', 'news', 'article', '']
+    slugs = list(filter(None, slugs))
+
+    page_slug = slugs[-1]
+    return page_slug
+
+
+# This function is useful if a content person has moved a page
+# around the tree, but hasn't changed the page slug
+# eg: /news/proteas-win-again/ => /sport/proteas-win-again/
```

Comment:
I understand that. The comment in the code, made me understand that the content person was expected to change the slug.

I think your above comment explains it better.
```
if they move the page, the page url changes due to the parent nodes changing, and therefore page can be found on its slug
```